src/components/activityEventElems/AddToBalanceEventElem.tsx

```diff
@@ -1,19 +1,14 @@
 import { t } from '@lingui/macro'
 import ETHAmount from 'components/currency/ETHAmount'
 import RichNote from 'components/RichNote'
-import { AddToBalanceEvent } from 'models/subgraph-entities/vX/add-to-balance-event'
 
+import { ProjectEventsQuery } from 'generated/graphql'
 import { ActivityEvent } from './ActivityElement'
 
 export default function AddToBalanceEventElem({
   event,
 }: {
-  event:
-    | Pick<
-        AddToBalanceEvent,
-        'amount' | 'timestamp' | 'from' | 'note' | 'id' | 'txHash' | 'terminal'
-      >
-    | undefined
+  event: ProjectEventsQuery['projectEvents'][0]['addToBalanceEvent']
```

Comment:
we now use generated types instead of the custom subgraph type with Pick<...>

by using the `ProjectEventsQuery[...][...]` type instead of the generated `AddToBalanceEvent` type, we can ensure this component's typing is directly dependent on the entity keys included in the ProjectEventsQuery graphql declaration